### PR TITLE
OCPBUGS-19358: Allow GIT_LFS_SKIP_SMUDGE environment variable

### PR DESCRIPTION
**Pull request by coreydaley**
Since introducing the 'git-lfs' package into the build container, we have had customers that have an issue with the git-lfs automatically being run and would like the option to disable this functionality via the 'GIT_LFS_SKIP_SMUDGE' environment variable.

### CODE REVIEW

**Review by adambkaplan — APPROVED** [review]
/approve

...though not sure if I still have "approver" rights here. Would like to see the godoc updated before merging.

File: build/v1/consts.go
Comment (coreydaley) [reply build/v1/consts.go]: Updated comment, ptal

**Review by adambkaplan — APPROVED** [review]
/lgtm

@openshift/api-reviewers please review this update.

**Pull request merged**
